operations/encryption/Overview.md

```diff
@@ -0,0 +1,160 @@
+# Kubernetes Encryption
```

Comment:
If this is specifically for Kubernetes encryption, would it make more sense for this to be located in the `operations/kubernetes/encryption/` folder? And usually for the main file on a topic, we would name it README.md (and create a symlink for index.md pointing to it).

Alternatively, if we want this in the general encryption folder because we anticipate grouping it with other, non-k8s encryption topics, then our convention is to have the file name match its main header. So in this case, it would be Kubernetes_Encryption.md.

Reply:
I can change it to whatever makes sense. I'll swap it out for the latter. I was mostly following some other conventions I had seen.